### ISSUE TITLE
usage inTypescript 

### ISSUE THREAD

**Issue by 95deepan** [ddsol/mp3-duration#9]
How to use this module in typescript ? I got the following error when I imported mp3-duration in this way:
`import * as mp3 from 'mp3-duration'`   **Error:** expecting a function but got [object Undefined]

How can I import the function directly ? Are there any methods to import from mp3-duration to get the duration of an mp3 file?

**Comment by kaceman** [comment]
Did you find any solution for this problem?

**Comment by 95deepan** [comment]
No, But [Ioinc-Audio](https://github.com/arielfaur/ionic-audio) had satisfied all my needs.

**Comment by kaceman** [comment]
thank you

**Comment by Borewit** [comment]
[music-metadate](https://github.com/Borewit/music-metadata) supports TypeScript and can be used with async/await.

Let me know if you need something else.
